gwlfe/WriteOutputFiles.py

```diff
@@ -703,6 +712,59 @@ def WriteOutput(z):
     if z.WxYrEnd > SumWxYrEnd:
         SumWxYrEnd = z.WxYrEnd
 
+    # Which land use sources to include in the totals.
+    sources = (0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12)
+
+    # ha
+    AreaTotal = sum(z.Area[l] for l in sources)
+
+    # kg
+    SumSed = sum(z.AvLuSedYield[l] for l in sources) * TONNE_TO_KG
+    SumSed += z.AvStreamBankErosSum
+
+    # kg
+    SumNitr = sum(z.AvLuTotNitr[l] for l in sources)
+    SumNitr += z.AvStreamBankNSum
+    SumNitr += z.AvAnimalNSum * z.RetentFactorN * (1 - z.AttenN)
+    SumNitr += z.AvGroundNitrSum * z.RetentFactorN * (1 - z.AttenN)
+    SumNitr += YrPointNitr * z.RetentFactorN * (1 - z.AttenN)
+    SumNitr += z.AvSeptNitr * z.RetentFactorN * (1 - z.AttenN)
+
+    # kg
+    SumPhos = sum(z.AvLuTotPhos[l] for l in sources)
+    SumPhos += z.AvStreamBankPSum
+    SumPhos += z.AvAnimalPSum * z.RetentFactorP * (1 - z.AttenP)
+    SumPhos += z.AvGroundPhosSum * z.RetentFactorP * (1 - z.AttenP)
+    SumPhos += YrPointPhos * z.RetentFactorP * (1 - z.AttenP)
+    SumPhos += z.AvSeptPhos * z.RetentFactorP * (1 - z.AttenP)
+
+    # m^3/year
+    MeanFlow = (z.AvStreamFlowSum * CM_TO_M) * (AreaTotal * HA_TO_M2)
+
+    # Find index of month with lowest mean flow.
+    LowFlowMonth = z.AvStreamFlow.tolist().index(min(z.AvStreamFlow))
+
+    # m^3/year
+    MeanLowFlow = (z.AvStreamFlow[LowFlowMonth] * CM_TO_M) * (AreaTotal * HA_TO_M2)
+
+    # m^3/second
+    MeanFlowPS = MeanFlow * 31536000
```

Comment:
If this is per second, should this be a division instead? 
It currently produces:
```
MeanFlow: 20496145.621219214 m3/yr
MeanFlowPerSecond: 646366448310769.12 m3/s
```
which doesn't seem right.